packages/web3-react/src/constants/supportedConnectors.tsx

```diff
@@ -15,34 +15,12 @@ GNU Lesser General Public License for more details.
 You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
-import { ReactNode } from 'react'
+import { Connector } from '../types'
 import Logos from './../assets/logos'
 
-let supportedConnectors: {
-  id: string
-  name?: string
-  shortName?: string
-  logos: {
-    default: ReactNode
-    transparent?: ReactNode
-    connectorButton?: ReactNode
-    qrCode?: ReactNode
-    appIcon?: ReactNode
-    mobile?: ReactNode
-  }
-  logoBackground?: string
-  scannable?: boolean
-  extensions?: { [key: string]: string }
-  appUrls?: { [key: string]: string }
-  extensionIsInstalled?: () => any
-  defaultConnect?: () => any
-}[] = []
```

Comment:
I extracted this into a type in the types file so that it can be reused across the project!